advanced/keyboard-shortcuts.md

```diff
@@ -11,47 +11,51 @@ keywords: keyboard shortcuts
+| Undo                                         | Ctrl+Z                | Command+Z                   | core                                                       |
+| Header 1                                     | Alt+Shift+1           | Ctrl+Option+1               | core                                                       |
+| Header 2                                     | Alt+Shift+2           | Ctrl+Option+2               | core                                                       |
+| Header 3                                     | Alt+Shift+3           | Ctrl+Option+3               | core                                                       |
+| Header 4                                     | Alt+Shift+4           | Ctrl+Option+4               | core                                                       |
+| Header 5                                     | Alt+Shift+5           | Ctrl+Option+5               | core                                                       |
+| Header 6                                     | Alt+Shift+6           | Ctrl+Option+6               | core                                                       |
+| Paragraph                                    | Alt+Shift+7           | Ctrl+Option+7               | core                                                       |
+| Div                                          | Alt+Shift+8           | Ctrl+Option+8               | core                                                       |
+| Address                                      | Alt+Shift+9           | Ctrl+Option+9               | core                                                       |
+| Focus to menu bar                            | Alt+F9                | Option+F9                   | core                                                       |
+| Focus to toolbar                             | Alt+F10               | Option+F10                  | core                                                       |
+| Focus to element path                        | Alt+F11               | Option+F11                  | core                                                       |
+| Focus to contextual toolbar                  | Ctrl+F9               | Ctrl+F9                     | core                                                       |
+| Open the popup menu on split toolbar buttons | Shift+Enter           | Shift+Enter                 | core                                                       |
+| Open the help dialog                         | Alt+0                 | Option+0                    | [help]({{ site.baseurl }}/plugins/help/)                   |
+| Insert link                                  | Ctrl+K                | Command+K                   | [link]({{ site.baseurl }}/plugins/link/)                   |
+| Toggle Fullscreen                            | Ctrl+Shift+F          | Ctrl+Shift+F                | [fullscreen]({{ site.baseurl }}/plugins/fullscreen/)       |
```

Comment:
This is incorrect, it should be `Command+Shift+F` for Mac.

Reply:
Fixed by 467979ea69821ffb840fd6066847fe75d513b82b